### ISSUE TITLE
Release and publish to NPM

### ISSUE THREAD

**Issue by Download** [chaijs/deep-eql#7]
Hi, I came here because I am investigating double dependencies in my project. 

```
├─┬ chai@3.5.0
│ ├── assertion-error@1.0.1
│ ├─┬ deep-eql@0.1.3
│ │ └── type-detect@0.1.1
│ └── type-detect@1.0.0
```

As you can see `type-detect` is included twice because `deep-eql` still depends on 0.x. I see you already upgraded it to 1.0.0, but it hasn't been published to NPM. Could you do that please?

**Comment by Download** [comment]
it would be a good thing if this lib went to version 1.x BTW, as NPM does not deal so gracefully with versions < 1.0.0... Also, as this lib is 3 years old and has been downloaded 1.5M times just this month, it's probably safe to declare it stable  :)

**Comment by keithamus** [comment]
@Download thanks for the issue and your feedback.

I'll be putting some work into deep-eql on Thursday to address some issues with the project, at which point it'll be bumped to 1.0 and chai 4.0.0 will include it.

**Comment by Download** [comment]
That's fantastic news @keithamus ! Thanks!

**Comment by keithamus** [comment]
We've released deep-eql 1.0, which will in turn be released with chai 4 soon, which will have a matching version of type-detect. Thanks for this issue @Download.
